### PR TITLE
Add Quick Start Guide to the readme

### PR DESCRIPTION
**Pull request by matthiasmoritz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,29 @@ You then need to load the plugin. In `boostrap.php`, something like:
...
+$this->addBehavior('Muffin/Tags.Tag');
+```
+
+and in the view
+
+```php
+echo $this->Form->input('tags');
+```
+
+Enjoy tagging
```
Comment (jadb) [inline README.md]: Exclamation point (`!`), common! :wink:

File: README.md [hunk 2]
```diff
@@ -25,6 +25,29 @@ You then need to load the plugin. In `boostrap.php`, something like:
...
+bin/cake migration migrate -plugin Muffin/Tags
+```
+
+add the behavior 
+
+```php
+$this->addBehavior('Muffin/Tags.Tag');
+```
+
+and in the view
```
Comment (jadb) [inline README.md]: Same

File: README.md [hunk 3]
```diff
@@ -25,6 +25,29 @@ You then need to load the plugin. In `boostrap.php`, something like:
...
+## Quick Start Guide
+
+You need to add the column *tag_count* to the taggable table
+
+then migrate the tables for the plugin 
+```
+bin/cake migration migrate -plugin Muffin/Tags
+```
+
+add the behavior
```
Comment (jadb) [inline README.md]: Punctuation.. hehe

File: README.md [hunk 4]
```diff
@@ -25,6 +25,29 @@ You then need to load the plugin. In `boostrap.php`, something like:
 
 {{@TODO documentation}}
 
+## Quick Start Guide
+
+You need to add the column *tag_count* to the taggable table
+
+then migrate the tables for the plugin
```
Comment (jadb) [inline README.md]: Column (`:`) at the end maybe?

File: README.md [hunk 5]
```diff
@@ -25,6 +25,29 @@ You then need to load the plugin. In `boostrap.php`, something like:
 
 {{@TODO documentation}}
 
+## Quick Start Guide
+
+You need to add the column *tag_count* to the taggable table
```
Comment (jadb) [inline README.md]: Maybe mention it could be customized real quick?

**Comment by jadb** [comment]
Apart from punctuations in some places, this looks great! Thanks!! :+1:

**Comment by matthiasmoritz** [comment]
just saw, that there are typos in the migration 
+bin/cake migration migrate -plugin Muffin/Tags

shoud be 
bin/cake migrations migrate -p Muffin/Tags

**Comment by jadb** [comment]
@matthiasmoritz good catch!

File: README.md (hunk 5 above)
Comment (matthiasmoritz) [inline README.md]: How can it be customized?

Comment (jadb) [inline README.md]: When loading the behavior:

https://github.com/UseMuffin/Tags/blob/master/src/Model/Behavior/TagBehavior.php#L36

You can define more than one field to track different types of tags and/or assign a custom one.

File: README.md [hunk 6]
```diff
@@ -25,6 +25,29 @@ You then need to load the plugin. In `boostrap.php`, something like:
 
 {{@TODO documentation}}
```
Comment (jadb) [inline README.md]: Maybe also replace that by the composer's package description:

```
Tags plugin for [CakePHP 3.x][cakephp].
```

File: README.md (hunk 5 above)
Comment (matthiasmoritz) [inline README.md]: Ah, I see. But as this is a Quick Start Guide, i would not mention it. This is a issue for the (still missing) docs :)

**Comment by jadb** [comment]
Thanks!

**Pull request merged**